client/src/App.js

```diff
@@ -40,34 +34,34 @@ function App() {
           />
         </Navbar.Brand>
 
-        <Nav className='mr.auto'>
-          <Link to='/dash'>Dash</Link>
-          <Link to='/categories'>Categories</Link>
-        </Nav>
-        <Navbar.Collapse className="justify-content-end">
+        <Navbar.Collapse className='justify-content-end'>
           <Nav className='mr.auto'>
             {document.cookie ? (
-              <Button onClick={logout}>Logout</Button>
+              <>
+                <Nav className='mr.auto'>
+                  <Link to='/dash'>Dash</Link>
+                  <Link to='/categories'>Categories</Link>
+                  <Button onClick={logout} href='/login'>Logout</Button>
+                </Nav>
+              </>
             ) : (
-                <>
-                  <Link to='/login'>Login</Link>
-                  <Link to='/register'>Register</Link>
-                </>
-              )}
+              <>
+                <Link to='/login'>Login</Link>
+                <Link to='/register'>Register</Link>
+              </>
+            )}
```

Comment:
If you're going to put the nav's inside the ternary, then put it here too, so we don't end up not having a navbar